dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -363,11 +642,23 @@ std::vector<std::string> LocalAdmissionController::handleTokenBucketsResp(
         }
 
         int64_t trickle_ms = granted_token_bucket.trickle_time_ms();
-        RUNTIME_CHECK(trickle_ms >= 0);
+        RUNTIME_CHECK_MSG(trickle_ms >= 0, "{}, expect trickle_ms >= 0, but got: {}", err_msg, trickle_ms);
```

Comment:
There could be multiple resource group in one `TokenBucketsResponse`, throwing an exception when handling one resource group will let tiflash can not handle the resp of other resource group.
What about just log an error for this resource group and continue handle the next resource group?